### PR TITLE
[Feature] Update icons

### PR DESCRIPTION
**Pull request by aleksandr-tikhonov**
Hi, this PR combines icons and illustrations just in one folder - **Assets**. 

**Please check that your screen has correct images**

If you need an image in your feature, please see it in this asset first, if it's not here, then add new image from design specs to this asset.

Structure of the asset:
- Common (for icons)
- Illustrations (for pictures)

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewController.swift [hunk 1]
```diff
@@ -155,9 +155,9 @@ extension ExposureDetectionViewController {
 
 	private func updateCloseButton() {
 		if state.isTracingEnabled {
-			closeImage.image = UIImage(named: "exposure-detection-close-contrast")
+			closeImage.image = UIImage(named: "Icons - Close")
 		} else {
-			closeImage.image = UIImage(named: "exposure-detection-close")
+			closeImage.image = UIImage(named: "Icons - Close")
```
Comment (EisvogelSAP) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewController.swift:160]: When tracing is enabled the contrast icon MUST be used.

**Review by eisvogel — CHANGES_REQUESTED** [review]
Why are we using inconsistent naming schemes all over the place?

**Review by johannesrohwer — APPROVED** [review]
Exposure Submission looks good to me (regarding icons/images)

File: src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewController.swift (hunk 1 above)
Comment (aleksandr-tikhonov) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewController.swift]: fixed

**Comment by aleksandr-tikhonov** [comment]
> Why are we using inconsistent naming schemes all over the place?

I just copied files form design specs. I think, naming convention for icons isn't so important and we can not waste time on this. I sure, the naming is not a problem at all.

File: src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewController.swift (hunk 1 above)
Comment (EisvogelSAP) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewController.swift]: Did you push changes? For me it is still wrong.

Comment (aleksandr-tikhonov) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewController.swift]: yep, let me check

File: src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewController.swift [hunk 2]
```diff
@@ -155,9 +155,9 @@ extension ExposureDetectionViewController {
 
 	private func updateCloseButton() {
 		if state.isTracingEnabled {
-			closeImage.image = UIImage(named: "exposure-detection-close-contrast")
+			closeImage.image = UIImage(named: "Icons - Close - Contrast")
```
Comment (aleksandr-tikhonov) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewController.swift:158]: This "Icons - Close - Contrast" image is just "Icons - Close" dark image

File: src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewController.swift (hunk 1 above)
Comment (aleksandr-tikhonov) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewController.swift]: Yes, I pushed changes

**Review by eisvogel — APPROVED** [review]

**Pull request merged**